core/src/test/scala/unit/kafka/server/KafkaApisTest.scala

```diff
@@ -126,6 +132,93 @@ class KafkaApisTest {
     )
   }
 
+  @Test
+  def testAuthorize(): Unit = {
+    val authorizer: Authorizer = EasyMock.niceMock(classOf[Authorizer])
+
+    val operation = AclOperation.WRITE
+    val resourceType = ResourceType.TOPIC
+    val resourceName = "topic-1"
+    val requestHeader = new RequestHeader(ApiKeys.PRODUCE, ApiKeys.PRODUCE.latestVersion(),
```

Comment:
Nit: is the `()` needed here? Same question in the next test.

Reply:
Right, it is not.